packages/mutation-testing-elements/src/components/mutation-test-report-file.ts

```diff
@@ -0,0 +1,95 @@
+import { LitElement, html, property, customElement, css, PropertyValues } from 'lit-element';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import hljs from 'highlight.js/lib/highlight';
+import javascript from 'highlight.js/lib/languages/javascript';
+import scala from 'highlight.js/lib/languages/scala';
+import java from 'highlight.js/lib/languages/java';
+import cs from 'highlight.js/lib/languages/cs';
+import typescript from 'highlight.js/lib/languages/typescript';
+import { MutationTestReportMutantComponent } from './mutation-test-report-mutant';
+import { MutantFilter } from './mutation-test-report-file-legend';
+import { bootstrap, highlightJS } from '../style';
+import { FileResultModel } from '../model';
+import { renderCode } from '../lib/helpers';
+
+hljs.registerLanguage('javascript', javascript);
+hljs.registerLanguage('typescript', typescript);
+hljs.registerLanguage('cs', cs);
+hljs.registerLanguage('java', java);
+hljs.registerLanguage('scala', scala);
+
+@customElement('mutation-test-report-file')
+export class MutationTestReportFileComponent extends LitElement {
+
+  @property()
+  public model!: FileResultModel;
+
+  public static styles = [
+    highlightJS,
+    bootstrap,
+    css`
```

Comment:
Shouldn't styles like this be in a separate css (or sass) file, so that Webpack can properly optimize them (and, if we would want, use magic like autoprefixer and such with it)

Reply:
Good catch! Both `bootstrap` and `highlightjs` are separated into different files.  We should do the same for custom files.